Composer/packages/client/src/recoilModel/atoms/appState.ts

```diff
@@ -30,6 +30,18 @@ export type CurrentUser = {
   sessionExpired: boolean;
 };
 
+export type PageMode =
+  | 'home'
```

Comment:
is there a reason why we are not using enums

PageMode = { home = 'home', design = 'design'....}?

Reply:
Good idea - I'll switch that over.

Reply:
I'm more in favor of union types. It makes debugging easier and you don't have to import them. We are in typescript land, why do you say it's error prone?

Reply:
It's just a personal preference. Since its a string enum atleast the debugging part would be the same. The reason i prefer enums is there are cases where I needed to enumerate over the enum. But I think union type or enum are fine in this case.

Reply:
After considering this, I think I'm going to stick with the union type. It really is just a type annotation that gets transpiled out, and we only ever need to use it for a single Recoil atom (and whatever reads from it).